study.md

```diff
@@ -137,7 +151,9 @@ Using JavaScript's Array methods, set a variable named `hankKeys` to `hank`'s
 keys.  Additionally, set a variable named `hankValues` to `hanks`'s values.'
 
 ```javascript
-// your answer here
+let hankKeys = hank.keys
```

Comment:
This will only work if `hank` has a property called `keys` on it, which it doesn't.